Cargo.toml

```
@@ -101,16 +102,14 @@ name = "schnorr_benchmarks"
 harness = false
 
 [features]
-default = ["std", "u64_backend", "getrandom"] # "rand"
+default = ["alloc", "getrandom", "cfg-if"] # "rand"
```

:wrench: Don't think `cfg-if` needs to be added.
Looking more thuroughly I don't think `cfg-if` is used at all in this repo
```suggestion
default = ["alloc", "getrandom"] # "rand"
```

---

It turns out `cfg-if` is actually needed to fix the `std` feature build errors. See [bd8b22c](https://github.com/mobilecoinfoundation/schnorrkel/pull/4/commits/bd8b22cdbb62a9dabd93e879e1c2ecf86f10af68)

---

Oh actually, i see what you're saying. Just removed cfg-if as a feature since it's always needed.